tests/opd/hash.txt

```diff
@@ -1,3 +1,2 @@
 opd -H ff0109
-9c2522e2c34f46724c6a4e95f460ced3eb30b72eb6b19a8cc832681fd75aabab
```

Comment:
how come the hash changed here?

Reply:
Good question :)
This is because we updated `opd -H` to use `sha256tree` instead of `sha256(bytes(prog))`